velox/docs/functions/spark/map.rst

```diff
@@ -6,18 +6,13 @@ Map Functions
 
     Returns value for given ``key``, or ``NULL`` if the key is not contained in the map.
 
-.. spark:function:: map() -> map(unknown, unknown)
+.. spark:function:: map(K, V, K, V, ...) -> map(K,V)
 
-    Returns an empty map. ::
+    Returns a map created using the given key/value pairs. Keys are not allowed to be null. ::
```

Comment:
Does this function allow zero arguments? If so, would you clarify that and include an example?

Reply:
Zero argument is not allowed, and we don't need this for spark, as `map()` is always handled by spark optimizer and converted to `{}` literals.

Reply:
Got it. Perhaps, update PR description to say that we are removing map() function.